assets/gdrive/gdrive_relations.json

```diff
@@ -1,214 +1,139 @@
 {
   "relations": [
     {
-      "object_type": "identity",
-      "object_id": "1",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "luke@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "2",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "leia@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "3",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "han@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "4",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "r2d2@droids.yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "3",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "c3po@droids.yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "luke@yavin4base.mil",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "luke@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "2",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "leia@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "leia@yavin4base.mil",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "han@yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "r2d2@yavin4base.mil",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "r2d2@droids.yavin4base.mil"
-    },
-    {
-      "object_type": "identity",
-      "object_id": "c3po@yavin4base.mil",
-      "relation": "identifier",
-      "subject_type": "user",
-      "subject_id": "c3po@droids.yavin4base.mil"
-    },
-    {
-      "object_type": "group",
-      "object_id": "everyone",
-      "relation": "member",
-      "subject_type": "user",
-      "subject_id": "luke.yavin4base.mil"
-    },
-    {
-      "object_type": "group",
-      "object_id": "everyone",
-      "relation": "member",
+      "object_type": "folder",
+      "object_id": "root",
+      "relation": "owner",
       "subject_type": "user",
-      "subject_id": "leia.yavin4base.mil"
+      "subject_id": "beth@the-smiths.com"
     },
     {
-      "object_type": "group",
-      "object_id": "everyone",
-      "relation": "member",
-      "subject_type": "user",
-      "subject_id": "han.yavin4base.mil"
+      "object_type": "doc",
+      "object_id": "secrets.txt",
+      "relation": "parent",
+      "subject_type": "folder",
+      "subject_id": "root"
     },
     {
-      "object_type": "group",
-      "object_id": "everyone",
-      "relation": "member",
+      "object_type": "doc",
+      "object_id": "secrets.txt",
```

Comment:
since this is in the root folder, I wonder if we want to call it "groceries.txt" and have a wildcard (user:*) relationship for it?

Reply:
Only Beth has access to the root folder so I wanted to demonstrate that.
But I think it would be a great idea to add a second file there and explicitly grant `user:*` access to it.

Reply:
I added a groceries.txt file at the root, and made `user:*` viewer.
Now everyone has read access to the file but only Beth can write to it because she's the owner of `root`.